main.rb

```diff
@@ -0,0 +1,9 @@
+require './lib/library_app.rb'
```

Comment:
Let's call this runner.rb or boot.rb. Such names are used for files that startup the application